data/sql/updates/pending_db_world/rev_1605009324701425500.sql

```diff
@@ -0,0 +1,118 @@
+(@NPC_JOSEPHINE,0,10,11,61,0,100,0,0,0,0,0,11,50711,0,0,0,0,0,1,0,0,0,0,0,0,0, 'Crusader Josephine - On spellhit Quest Credit - Cast Strip Aura Josephine 01'),
+(@NPC_JOSEPHINE,0,11,12,61,0,100,0,0,0,0,0,86,50699,0,12,1,0,0,1,0,0,0,0,0,0,0, 'Crusader Josephine - On spellhit Quest Credit - Crosscast Kill Credit Josephine'),
+(@NPC_JOSEPHINE,0,12,13,61,0,100,0,0,0,0,0,86,50712,0,12,1,0,0,1,0,0,0,0,0,0,0, 'Crusader Josephine - On spellhit Quest Credit - Crosscast Strip Aura Josephine'),
+(@NPC_JOSEPHINE,0,13,14,61,0,100,0,0,0,0,0,29,0,0,0,0,0,0,0,0,0,0,0,0,0,0, 'Crusader Josephine - On spellhit Quest Credit - Stop follow'),
+(@NPC_JOSEPHINE,0,14,15,61,0,100,0,0,0,0,0,1,0,0,0,0,0,0,1,0,0,0,0,0,0,0, 'Crusader Josephine - On spellhit Quest Credit - Say text'),
+(@NPC_JOSEPHINE,0,15,16,61,0,100,0,0,0,0,0,53,0,@NPC_JOSEPHINE,0,0,0,0,1,0,0,0,0,0,0,0, 'Crusader Josephine - On spellhit Quest Credit - Start WP movement'),
+(@NPC_JOSEPHINE,0,16,0,61,0,100,0,0,0,0,0,83,1,0,0,0,0,0,1,0,0,0,0,0,0,0, 'Crusader Josephine - On spellhit Quest Credit - Remove npcflag'),
+(@NPC_JOSEPHINE*100,9,0,0,0,0,100,0,0,0,0,0,1,1,0,0,0,0,0,1,0,0,0,0,0,0,0, 'Crusader Josephine script - Say text'),
+(@NPC_JOSEPHINE*100,9,1,0,0,0,100,0,0,0,0,0,22,2,0,0,0,0,0,1,0,0,0,0,0,0,0, 'Crusader Josephine script - Set phase 2'),
+(@NPC_JOSEPHINE*100,9,2,0,0,0,100,0,0,0,0,0,29,0,0,0,0,0,0,0,0,0,0,0,0,0,0, 'Crusader Josephine script - Stop follow'),
+(@NPC_JOSEPHINE*100,9,3,0,0,0,100,0,2000,2000,0,0,37,0,0,0,0,0,0,1,0,0,0,0,0,0,0, 'Crusader Josephine script - Die');
+
+DELETE FROM `waypoints` WHERE `entry` IN (@NPC_JONATHAN,@NPC_LAMOOF,@NPC_JOSEPHINE);
+INSERT INTO `waypoints` (`entry`,`pointid`,`position_x`,`position_y`,`position_z`) VALUES
+(@NPC_JONATHAN,1,5257.454,-3500.14,291.6933),
+(@NPC_JONATHAN,2,5253.089,-3516.885,291.6786),
+(@NPC_JONATHAN,3,5255.452,-3523.673,291.6932),
+(@NPC_JONATHAN,4,5262.733,-3527.41,291.6934),
+(@NPC_JONATHAN,5,5261.445,-3528.885,291.6929),
+
+(@NPC_LAMOOF,1,5257.454,-3500.14,291.6933),
+(@NPC_LAMOOF,2,5253.089,-3516.885,291.6786),
+(@NPC_LAMOOF,3,5255.452,-3523.673,291.6932),
+(@NPC_LAMOOF,4,5260.741,-3525.38,291.69343),
+(@NPC_LAMOOF,5,5259.029,-3527.101,291.4913),
+
+(@NPC_JOSEPHINE,1,5257.454,-3500.14,291.6933),
+(@NPC_JOSEPHINE,2,5253.089,-3516.885,291.6786),
+(@NPC_JOSEPHINE,3,5257.784,-3521.994,291.6931),
+(@NPC_JOSEPHINE,4,5256.293,-3523.494,291.6933);
+
+DELETE FROM `creature_text` WHERE `creatureid` IN (@NPC_JONATHAN,@NPC_LAMOOF,@NPC_JOSEPHINE) AND `groupid`=1;
+INSERT INTO `creature_text` (`creatureid`,`groupid`,`id`,`text`,`type`,`language`,`probability`,`emote`,`duration`,`sound`,`comment`) VALUES
+(@NPC_JONATHAN,1,0,'<sigh>... This is the end of me.',12,0,100,1,0,0,''),
+(@NPC_LAMOOF,1,0,'<sigh>... This is the end of me.',12,0,100,1,0,0,''),
+(@NPC_JOSEPHINE,1,0,'Ohhh, I... cannot go on....',12,0,100,1,0,0,'');
```

Comment:
Should there be four dots at the end?

Reply:
https://youtu.be/aPn9h39rwHw?t=286

In the gossip are 4 dots, so i think its correct
Searched it too, cause, 4 dots is weird.. But seems OK so

Reply:
It's weird.
In this video from WoD she doesn't open a gossip when he right clicks
https://youtu.be/Evx9-8J1Z8w?t=228

Reply:
Nvm I found this from WotLK https://www.youtube.com/watch?v=lLOpUvAxjw0 it seems to be correct in the gossip